src/redux/books/books.js

```diff
@@ -0,0 +1,24 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+export const addBook = (title, author) => ({
+  type: ADD_BOOK,
+  title,
+  author,
+});
+
+export const removeBOok = (title) => ({
+  type: REMOVE_BOOK,
+  title,
+});
+
+export const bookReducer = (state = [], action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, { title: action.title, author: action.author }];
```

Comment:
- [OPTIONAL] I highly recommend you follow the best structure by adding the payload which will hold of the new added book.

`
export const addBook = (book) => ({
  type: ADD_BOOK,
  payload: book
});
`

` case ADD_BOOK:
      return [...state, action.payload];`